python/TestHarness/TestHarness.py

```diff
@@ -393,40 +393,37 @@ def canPrint(self, tester):
 
         return True
 
-    # Format the caveats contained in tester so they are easy to read when printed
+    # Format results the way we want it
+    # TODO: refactore. It seems we only need it so the word FAILED
```

Comment:
How involved would this refactor (not refactore) be? Something that would be suitable for this PR?

Reply:
This is something I will certainly be addressing in #10456